frontend/src/app/edit-claim/edit-claim.component.ts

```diff
@@ -47,9 +47,13 @@ export class EditClaimComponent implements OnInit {
     }
   }
 
-  deleteFile(hash: string): void {
+  deleteFile(filename: string): void {
     // FIXME: This is not correct
-    this.service.deleteFile(hash, this.vin).subscribe();
+    this.service.deleteFile(filename, this.vin).subscribe(
```

Comment:
subscribe is apparently deprecated, so be careful here.

Reply:
will look at the current way to handle this next PR.